vars/fhcapComponentUpdate.groovy

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/groovy
+
+def call(body) {
+    def config = [:]
+    body.resolveStrategy = Closure.DELEGATE_FIRST
+    body.delegate = config
+    body()
+
+    String componentName = config.componentName
+    String componentVersion = config.componentVersion
+    String componentBuild = config.componentBuild
+    String sourceBranch = config.sourceBranch ?: ''
+    String targetBranch = config.targetBranch ?: ''
+    String changeUrl = config.changeUrl ?: ''
+    boolean useGeneratedJobs = true
```

Comment:
Need a way to make this configurable without having to set it in the Jenkinsfile.